.gitlab-ci.yml

```
@@ -590,6 +620,19 @@ workflow:
       DSD_REPOSITORY: ci/datadog-agent/dogstatsd-release
       IMG_REGISTRIES: internal-aws-ddbuild
 
+.on_deploy_a7_internal_rc_manual:
+  - <<: *if_not_version_7
+    when: never
+  - <<: *if_not_deploy
+    when: never
+  - <<: *if_rc_tag_on_beta_repo_branch
+    when: on_success
```

```suggestion
    when: manual
```
I suppose it's exactly the same as `on_deploy_a7_rc_manual` but the variables?

---

Yes oops